reference/client-access/python-sdk.md

```diff
@@ -0,0 +1,287 @@
+instance managed by Turso.
+
+In either case, the returned Client object exposes an identical API. The
+synchronous client is a thin wrapper around the asynchronous client.
+
+The examples on this page use the synchronous API. The equivalent async code can
+be easily derived from that by applying `async` and `await`.  For example, a
+synchronous call to the client's `execute()` method takes this form:
+
+```py
+with client:
+    result_set = client.execute("...")
+    # Work with query results
+```
+
+While an asynchronous call takes this form:
+
+```py
+async with client:
+    result_set = await client.execute("...")
+    # Work with query results
+```
+
+In both cases above, the `with` keyword is used to close the client after the
+work is complete. If you want to work with the client object across functions,
+you should call `close()` on it after you're done with the client to clean up
+the resources it uses.
+
+## Example data set
+
+All of the examples in this section assume tables and data established by these
+statements:
+
+```sql
+create table example_users (
+    uid text primary key,
+    email text
+);
+create table example_scores (
+    uid text,
+    level integer,
+    score integer,
+    primary key (uid, level)
+);
+
+insert into example_users values ('uid1', 'foo@bar.com');
+insert into example_users values ('uid2', 'baz@bar.com');
+insert into example_scores values ('uid1', 1, 100);
+insert into example_scores values ('uid1', 2, 95);
+```
+
+## Execute a single statement
+
+### SQL string argument
+
+You can execute a single statement with the `execute()` method of a Client
+object. It returns a [ResultSet](#resultset) object containing the results of
+the query.
+
+```py
+rs = client.execute("select * from users")
+
+# rs.columns == ('uid', 'email')
+# rs.rows[0] == ('uid1', 'foo@bar.com')
+# rs.rows[1] == ('uid2', 'baz@bar.com')
+```
+
+### Positional placeholders
+
+libSQL supports positional placeholders using the same syntax as SQLite. Pass
+two parameters, with the first (`stmt`) being the SQL string with placeholders,
+and the second (`args`) being an array of values to bind to the placeholders.
+
+```py
+rs = client.execute(
+    stmt="select score from example_scores where uid = ? and level = ?",
+    args=[ "uid1", 2 ]
+)
+
+# rs.columns == ('score',)
+# rs.rows[0] == (95,)
+```
+
+### Named placeholders
+
+libSQL supports positional placeholders using the same syntax as SQLite. Pass
+two parameters, with the first (`stmt`) must be the SQL string with
+placeholders, and the second (`args`) must be a dictionary whose keys match the
+names of the placeholders, and whose values will be bound to the placeholders.
+
+```py
+rs = client.execute(
+    stmt="insert into example_scores values (:uid, :level, :score)",
+    args={ "uid": "uid2", "level": 1, "score": 50 }
+)
+
+# rs.columns == ()
+# rs.rows == []
+# rs.rows_affected == 1
+```
+
+In the above example, there are placeholders for `uid`, `level`, and `score`
```

Comment:
Maybe this should be "the example above" instead of "the above example"?

Reply:
Either works in English.